scripts/checkLinks.mjs

```diff
@@ -0,0 +1,64 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const extractLinks = (content) => {
+  const linkRegex = /link:\s*'([^']+)'/g;
```

Comment:
I guess there is nothing to stop someone from using double quotes with links?

Reply:
good catch!

Reply:
I think this would also break the CI if there is a comment that contains a broken link?